bin/node/cli/tests/export_import_flow.rs

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg(unix)]
+
+use assert_cmd::cargo::cargo_bin;
+use std::{process::Command, fs, path::PathBuf};
+use tempfile::{tempdir, TempDir};
+use regex::Regex;
+use lazy_static::lazy_static;
+
+mod common;
+
+fn contains_error(stderr: &[u8]) -> bool {
+	let log_output = String::from_utf8_lossy(stderr).to_owned();
+    lazy_static! {
```

Comment:
Why are you using a lazy static here?

Reply:
Stole it from [regex's docs](https://docs.rs/regex/1.3.7/regex/#example-avoid-compiling-the-same-regex-in-a-loop) to avoid compiling the same regex multiple times. Definitely unnecessary optimization, so if you feel like we should remove it no worries :)

Reply:
Is `Regex` really needed here? Looks like `log_output.contains("Error")` would work just the same

Reply:
Haha thank you I didn't know this method existed on Strings! :)

Reply:
(I think the issue was more that the lazy_static macro was in the middle of a function. It's usually used on the top of the file near other statics)